lift/src/main/scala/bootstrap/liftweb/Boot.scala

```diff
@@ -22,5 +22,17 @@ class Boot {
 
     // Force the request to be UTF-8
     LiftRules.early.append(_.setCharacterEncoding("UTF-8"))
+    
+    def hasTrailingIndex_?(path: ParsePath): Boolean = path.partPath.lastOption.contains("index")
+
+    // Rewriting /index to /
+    LiftRules.statelessRewrite.append {

```

Comment:
I know, unfortunately [it's not customizable](https://github.com/lift/framework/blob/master/web/webkit/src/main/scala/net/liftweb/http/Req.scala#L542)